app/assets/stylesheets/subject.scss

```diff
@@ -1,4 +1,21 @@
 .dropdown-pane.is-open {
   width: 200px;
 
+
+}
+
+
+.subjects_index_container {
+  .title {
+    font-size: 2.5rem;
+  }
+
+  .subject_list {
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.